src/layout/section/SEALandingLayout/SEALandingLayout.style.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components'
+
+import { color, responsiveBreakpoints } from '../../../_utils/branding'
+
+const Grid = styled.div`
+  display: grid;
+  height: 100vh; //@todo: strech
+  grid-template-areas:
+    'figure'
+    'body';
+  grid-template-rows: minmax(33vh, 33vh) auto;
+  background-color: ${color.white};
+
+  @media (${responsiveBreakpoints.isMediaLarge}) {
+    grid-template-areas: 'body figure';
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: auto;
+  }
+`
+const Body = styled.div`
+  grid-area: body;
+
+  @media (${responsiveBreakpoints.isMediaLarge}) {
+    align-self: center;
+    justify-self: center;
+    max-width: 23.4375rem; //magical number: iPhone6 width
```

Comment:
I don't understand the comment about iPhone 6 width. We are in a media large media query, but the comment is about iPhone 6

Reply:
Yes. The design has used the max-size of the content as a mobile one. So no changes. This is the same size as 
`componentSizes.searchOverlayWidth`. It keeps happing over and over again in the app.